tox.ini

```diff
@@ -449,6 +451,25 @@ commands =
     ; pytest --nbmake tutorials/pandas-cookbook -p no:randomly -vvvv
 
 
+[testenv:single_container.launch]
+description = Launch a single backend container using the dockerfile
+changedir = {toxinidir}/packages
+allowlist_externals =
+    bash
```

Comment:
suggestion: can we accept NODE_NAME,  NODE_TYPE and PORT as env, (how we do it k8s.dev.deploy) so that users can set them instead of using the default. We can also set a label
```
--label orgs.openmined.syft="this is a syft worker container"
```
This label together with NODE_NAME can later be used to remove containers, volumes, etc.

Reply:
Resolved in [c03f06f](https://github.com/OpenMined/PySyft/pull/8777/commits/c03f06fb6c336b57ffbada66345c63ce3644c808)

Reply:
The label might be useful when we have multiple contianers, and to land all the containers.
I think for single containers, the node name would be unique identifier for landing.
We could omit labels for now?
I have added the remaining suggesstions.